autoload/fzf/vim.vim

```diff
@@ -41,7 +41,8 @@ if s:is_win
   else
     let s:bin.preview = fnamemodify(s:bin.preview, ':8')
   endif
-  let s:bin.preview = 'bash '.escape(s:bin.preview, '\')
+    let git_bash_bin = 'bash /mnt/'.substitute(s:bin.preview, '\([A-Z]\):', '\L\1\e', '')
+    let s:bin.preview = substitute(git_bash_bin, '\', '/', 'g')
```

Comment:
If you want to use mixed paths without using cygpath, ok. Problem is if it breaks the shortpath for Vim.